serve_test.go

```diff
@@ -561,3 +578,39 @@ func newTLSTestHelper() (*tlsTestHelper, error) {
 	keyPEM := pem.EncodeToMemory(&pem.Block{Type: "RSA PRIVATE KEY", Bytes: privateKeyBytes})
 	return &tlsTestHelper{cert: certPEM, key: keyPEM}, nil
 }
+
+func TestFlow(t *testing.T) {
+	newTestCtx := func(w *httptest.ResponseRecorder) *netHttpContext[any] {
+		return NewNetHTTPContext[any](
+			BaseRoute{},
+			w,
+			httptest.NewRequest("GET", "/", nil),
+			readOptions{},
+		)
+	}
+
+	t.Run("base", func(t *testing.T) {
+		e := NewEngine()
+		w := httptest.NewRecorder()
+		ctx := newTestCtx(w)
+		Flow(e, ctx, testController)
+		assert.Equal(t, http.StatusOK, w.Code)
+		assert.Equal(t, crlf(`{"ans":"Hello World"}`), w.Body.String())
+	})
+	t.Run("with nil return in ErrorHandler", func(t *testing.T) {
+		e := NewEngine(WithErrorHandler(func(err error) error { return nil }))
```

Comment:
What about a test with


```suggestion
		e := NewEngine(WithErrorHandler(nil)
```

Reply:
It'll throw, it's already tested elsewhere ;)